src/lib/import_export/csv_meta.hpp

```diff
@@ -16,12 +24,27 @@ struct CsvConfig {
   // If this is set to true, "4.3" will not be accepted as a value for a float column
   bool reject_quoted_nonstrings = true;
 
-  const char* meta_file_extension = ".meta";
-
   // Indicator whether the Csv follows RFC 4180. (see https://tools.ietf.org/html/rfc4180)
   bool rfc_mode = true;
 
   static constexpr const char* NULL_STRING = "null";
 };
 
+/*
+ * Predefined characters used for the csv file
+ */
+
+struct CsvMeta {
+  size_t chunk_size;
+  ParseConfig config;
+  std::vector<ColumnMeta> columns;
+
+  static constexpr const char* META_FILE_EXTENSION = ".json";
```

Comment:
Can we make this a `std::string`?

Reply:
nope

Reply:
we *could* make it a `string_view` if we really want to:

```c+11
using namespace std::string_view_literals;

static constexpr std::string_view META_FILE_EXTENSION = ".json"sv;
```

Reply:
Right. I'd prefer sticking to `char*` as of now (or `auto` for that matter, which would make it look more pretty), but if anyone has a strong opinion on that, I am fine with either.

Reply:
I would leave this as `char*` because it is consistent with all other random strings flying around.